.trivyignore

```diff
@@ -1 +1 @@
-CVE-2024-21538 exp:2024-12-31
\ No newline at end of file
+CVE-2024-21538 exp:2025-03-01
```

Comment:
to make trivy scanner pass again, this will be resolved by the earlier of the underlying issue to be merged in node, or upgrading to node 22